pom.xml

```diff
@@ -50,6 +50,12 @@
 			<roles><role>founder</role></roles>
 			<properties><id>dscho</id></properties>
 		</contributor>
+		<contributor>
+			<name>Deborah Schmidt</name>
+			<url>http://imagej.net/User:Frauzufall</url>
+			<roles><role>maintainer</role></roles>
```

Comment:
If you are a `maintainer`, then you are not a `<contributor>` but rather a `<developer>`.

Reply:
I would like to maintain this, yes. Do you agree or should I rather change my role?

Reply:
Wonderful—I wholeheartedly support additional maintainers on whichever components you feel are relevant to your work. Please see the [SciJava team roles](https://imagej.net/Team) and add yourself as a `<developer>` which whichever of those roles are appropriate.